LongWarOfTheChosen/Src/LW_PerkPack_Integrated/Classes/X2Action_Fire_ChainLightning.uc

```diff
@@ -0,0 +1,355 @@
+// the first shot is a standard projectile shot
```

Comment:
File header please. Also, I assume this is mostly copied from somewhere? If so, where?